lektor/imagetools.py

```diff
@@ -213,6 +213,11 @@ def documentname(self):
     def description(self):
         return self._get_string('Image ImageDescription')        
 
+    @property
+    def rotated(self):
+        # cd. <http://www.impulseadventure.com/photo/images/orient_flag.gif>
```

Comment:
Can you add a textual description as a docstring, instead of linking to an image? This image may not always exist at this URL, and a docstring is clearer anyway.